modules/ocf/manifests/serial_getty.pp

```diff
@@ -1,10 +1,12 @@
 class ocf::serial_getty {
-  service { 'serial-getty@ttyS0':
-    enable   => true,
-    provider => systemd,
-    require  => Package['systemd-sysv'],
-  } ~>
-  exec { 'systemctl start serial-getty@ttyS0':
-    refreshonly => true,
+  if str2bool($::is_virtual) or member(hiera('classes'), 'ocf_kvm') {
```

Comment:
Would it be better instead to directly add `ocf::serial_getty` to the hypervisor nodes in hiera?